modules/dists/CyclicDist.chpl

```diff
@@ -1069,7 +1069,7 @@ proc CyclicArr.doiBulkTransferFrom(Barg, viewDom)
           if debugCyclicDistBulkTransfer then
             writeln("B[",(...r1),"] ToDR A[",regionDest, "] ");
 
-          Barg._value.doiBulkTransferToDR(Dest.locArr[i].myElems[regionDest], Barg.domain[(...r1)]);
+          Barg._value.doiBulkTransferToDR(Dest.locArr[i].myElems[regionDest], {(...r1)});
```

Comment:
Can you tell me about this change?  (What was wrong before, what it fixes)

Reply:
Barg.rank is not necessarily equal to r1.size, and would cause a compilation failure under certain conditions. We already pass ``{(...r1)}`` in BlockDist, so doing the same here felt safe.